src/__tests__/App.test.js

```diff
@@ -0,0 +1,21 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import React from 'react';
+import store from '../Redux/store';
+import App from '../App';
+
+describe('App js testing', () => {
```

Comment:
- [ ] As your previous code reviewer mentioned about the test failing, it seems like the problem still persists and needs action to solve it. 

![](https://user-images.githubusercontent.com/62849872/266765003-76dcf6ed-75a8-4079-a21b-fcff8c27b13b.png)

Reply:
I don't know why is not working on your computer but it works on mine.
Please @skyv26, take a look at this short video.
https://www.loom.com/share/4c7b5681fe864c5081a7dd15b1dc3f84?sid=95d9841b-5788-473b-9600-e8195ca39253

Reply:
@MenendezON Please delete your `__snapshots__` folder and try it again and if everything seems fine re-submit your project again.

Reply:
this change has been done well